src/components/DroughtThresholds.vue

```diff
@@ -1771,10 +1804,88 @@ export default {
             */
           },
         }) 
+        if (i == 0) {
+          tl.to(`.${scrollClass}`, {
+            scrollTrigger: {
+              markers: this.marker_on,
+              trigger: `.${scrollClass}`,
+              start: "top 54%",
+              end: 99999,
+              toggleClass: {targets: ['.quietCircle', "#next"], className:"visible"}, // adds class to target when triggered
+              toggleActions: "restart none none reverse" 
+              /*
+              onEnter - scrolling down, start meets scroller-start
+              onLeave - scrolling down, end meets scroller-end
+              onEnterBack - scrolling up, end meets scroller-end
+              onLeaveBack - scrolling up, start meets scroller-start
+              */
+            },
+          })
+        }
+        if (i == 1) {
+          tl.to(`.${scrollClass}`, {
+            scrollTrigger: {
+              markers: this.marker_on,
+              trigger: `.${scrollClass}`,
+              start: "top 54%",
+              end: 99999,
+              toggleClass: {targets: "#prev", className:"visible"}, // adds class to target when triggered
+              toggleActions: "restart none none reverse" 
+              /*
+              onEnter - scrolling down, start meets scroller-start
+              onLeave - scrolling down, end meets scroller-end
+              onEnterBack - scrolling up, end meets scroller-end
+              onLeaveBack - scrolling up, start meets scroller-start
+              */
+            },
+          })
+        }
+        if (i == (containers.length-1)) {
```

Comment:
Hmm, yeah, I am super torn on if they should stay or disappear. Or if instead, there should be a clickable link or icon that is "reset" or "back to beginning" to restart progression.

Would it be useful for the references to listed after the last text instead of as a separate div at the end? I'll think on this and see what it looks like.

Reply:
yeah, that could be a good workaround if it seems to work/look okay